lib/quantum/job_broadcaster.ex

```diff
@@ -300,6 +300,33 @@ defmodule Quantum.JobBroadcaster do
     end
   end
 
+  def handle_cast(
+        {:run_job, name},
+        %State{
+          jobs: jobs,
+          debug_logging: debug_logging
+        } = state
+      ) do
+    debug_logging &&
+      Logger.debug(fn ->
+        "[#{inspect(Node.self())}][#{__MODULE__}] Running job #{inspect(name)} once"
+      end)
+
+    case Map.fetch(jobs, name) do
+      :error ->
+        {:noreply, [], state}
+
+      {:ok, job} ->
+        # Send event to telemetry incase the end user wants to monitor events
+        :telemetry.execute([:quantum, :job, :run], %{}, %{
```

Comment:
Telemetry events for execution are done here: https://github.com/quantum-elixir/quantum-core/blob/master/lib/quantum/executor.ex

I think this event is not needed, what do you think?

Reply:
If the telemetry event gets handled in the executor, then I agree it's not necessary here. I have made the adjustment in my next commit :)